src/EditorFeatures/Core/Implementation/IntelliSense/Completion/Controller_Commit.cs

```diff
@@ -36,6 +49,16 @@ private void Commit(CompletionItem item, TextChange textChange, Model model, cha
             // into us.  However, for now, we just hope that no such craziness will occur.
             this.StopModelComputation();
 
+            var waitResult = _waitIndicator.Wait(
```

Comment:
We don't want a wait indicator here.  Or, if it comes up it needs to be after a much longer delay.   Wait indicators shown during non-user-invoked actions (like typing) are extremely awful to the experience (worse than the blocking that they're trying to alleviate).